engine/src/main/java/org/terasology/rendering/nui/layers/mainMenu/SelectGameScreen.java

```diff
@@ -15,6 +15,7 @@
  */
 package org.terasology.rendering.nui.layers.mainMenu;
 
+import org.codehaus.plexus.util.FileUtils;
```

Comment:
Not sure why this plexus entry snuck in a few places. In this class it appears unused. Wrong import, later replaced with a more correct class, but not deleted?